Gemfile.lock

```diff
@@ -180,7 +193,7 @@ GEM
       sass (>= 3.3.0, < 3.5)
     compass-import-once (1.0.5)
       sass (>= 3.2, < 3.5)
-    compass-rails (4.0.0)
+    compass-rails (3.1.0)
```

Comment:
It seems a bit weird that we have to forceably downgrade `compass-rails` here for Rails 6...? :thinking:

I guess it's not that important, as we'll be switching to Webpacker soon, and I think we'll be dropping these sass gems completely at that point.

Reply:
nice, thanks for asking! I went back to it, removed the constraint in the Gemfile and typed  bundle update compass-rails. It works with v4, I must have left it in my initial attempts at getting bundle install to work with rails6. We are back to v4 :+1:

Reply:
Nice, yeah I had to do similar things getting bundler to install on previous upgrades :ok_hand: